### DISCUSSION TITLE
Beta10 - A big step forward..

### DISCUSSION THREAD

**Discussion by Pete3030** [apohl79/audiogridder#251] [General]
Just want to inform that Beta10 is running very smooth here - no problems occurred so far, even N4 works as expected. A big thank you for your hard work, Andreas!

**Comment by CaBuffalo** [comment]
It's been 24 hours and I don't had a chance to get a dump file. Beta 10 has been running very stably so far.

Keep testing.

Thanks for this great improvement, Andreas.
